pybuildkite/client.py

```diff
@@ -100,6 +100,20 @@ def put(self, url, body=None, headers=None, query_params=None):
 
         return self.request("PUT", url=url, query_params=query_params, body=body, headers=headers)
 
+    def delete(self, url, body=None, headers=None, query_params=None):
```

Comment:
DELETE is also being created https://github.com/pyasi/pybuildkite/pull/21 so we'll need to be smart about how we're merging.

CC: @savithakj